apps/mdn/mdn-aws/k8s/celery.workers.deploy.yaml.j2

```diff
@@ -3,11 +3,9 @@
             - /usr/local/bin/newrelic-admin
           args:
             - run-program
-            - ./manage.py
             - celery
+            - "--app=kuma.celery:app"
             - worker
-            - "--events"
```

Comment:
The new name for this is `--task-events`, but since we're dumping the `django-celery` `celerycam` service, I don't think workers need to emit events for tasks any longer.